app/controllers/ems_container_controller.rb

```diff
@@ -32,6 +32,11 @@ def ems_container_form_fields
     ems_form_fields
   end
 
+  def new
+    super
+    @ems_types -= [["Atomic", "atomic"], ["Atomic Enterprise", "atomic_enterprise"]]
```

Comment:
@durandom ah no... I entirely missed the `-` :smile: I would have probably used something more verbose:

```ruby
 @ems_types.reject! { |x| x[1].start_with?('atomic') }
```

anyway you can leave it as it is :+1: